red-hat-storage/cephci
# CEPH-11416 Pool Mirror - Try marking the secondary on the relationship for deletion, and observe the behaviour

Signed-off-by: Rajendra Khambadkar <rkhambad@rkhambad.remote.csb>

Polarion Link : https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-11416 

    Test Case Flow:
    1. Create an image in primary and perform IOs.
    2. enable pool mirroring on the image.
    3. verify for the pool mirror mode enabled on secondary.
    4. Move the secondary image to the trash.
    6. verify that unable to mark the secondary image for deletion, as image is locked.

success log : 
6.0 : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-9ox9w/
5.3 : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-1bzco/
